ports/metal-cpp/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "metal-cpp",
+  "version-date": "2022-05-19",
+  "description": "low overhead and header only C++ interface for Metal.",
```

```suggestion
  "description": "low overhead C++ interface for Metal.",
```

"header-only" is incorrect in the typical c++ sense -- you must compile an implementation to use this library.